examples/kitty/client/kitty.py

```diff
@@ -54,26 +54,24 @@ def show(self):
         fb.machine_fb_send(self.buf, self.width, self.height)
 
 
-# @alwin: all the vars highlighted are still (especially) dodgy
 ######################################################
```

Comment:
Maybe get rid of the ########### thing and move these up with the rest of the global variables